python/rikai/conf.py

```diff
@@ -0,0 +1,43 @@
+#  Copyright (c) 2021 Rikai Authors
+#
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+"""
+This is a temporary mechanism to wrap around pandas option machinery.
+We'll need a permanent solution later on for two main reasons:
+1. Users can accidentally clear options via pandas api
+2. We need better type handling to help bridge jvm-python communication (GH134)
+"""
+import pandas as pd
+from pandas._config.config import (register_option, get_option,
+                                   set_option, reset_option)
+options = pd.options
+
+CONF_RIKAI_CACHEURI = "rikai.cache_uri"
+CONF_PARQUET_BLOCK_SIZE = "parquet.block.size"
```

Comment:
do we expect to pass this to spark directly? 
otherwise, should we add `rikai` namespace to it?

Reply:
Hmm see the usage of this key. a little bit confused that this is not a rikai config key.

Reply:
yeah, i chose to stay consistent on the naming with the actual config key. I suppose we could have something like `rikai.spark.options.parquet.block.size` or something if we want to make it clear that it's being passed straight into Spark